roles/confluent.kafka_connect/tasks/main.yml

```diff
@@ -122,6 +122,7 @@
     secrets_file_group: "{{ kafka_connect_group }}"
     handler: restart connect distributed
   when: kafka_connect_secrets_protection_enabled|bool
+  no_log: "{{ mask_secrets|bool }}"
```

Comment:
what needs to be masked here?

Reply:
@nsharma-git ^^ can you include some log outputs with this flag set to true/false in the PR description?

Reply:
Hi @domenicbove , Since we have added condition to `secret_protection.yml`, this is actually not required. I have updated the code. Removed the `no_log` flag from all services while including the `secret_protection.yml`
For reference I have added the output of task `Create Kafka Broker Config with Secrets Protection` , I hope that should be good enough.